src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -199,15 +201,13 @@ class SaveInProgressIntro extends React.Component {
         unauthStartText,
       } = this.props;
       const unauthStartButton = (
-        <button
-          className="usa-button-primary"
+        <VaButton
           onClick={this.openLoginModal}
-          aria-label={ariaLabel}
-          aria-describedby={ariaDescribedby}
-          type="button"
-        >
-          {unauthStartText || UNAUTH_SIGN_IN_DEFAULT_MESSAGE}
-        </button>
+          label={ariaLabel}
+          // aria-describedby={ariaDescribedby}
```

Comment:
`va-button` doesn't include a method to add an `aria-describedby` to the button inside the shadow DOM. So instead of obsolete, I would say that the prop is missing & should be implemented - which is why I commented it out instead of removing it.